README.md

```diff
@@ -290,3 +290,9 @@ the name `SLACK_WEBHOOK_URL`, containing a url for [Incoming Webhooks](https://a
 - Local
   - [Swagger UI](http://localhost:8080/api/docs)
   - [JSON](http://localhost:8080/api/docs.json)
+
+## Note: Prototype Code
+
+This repository includes code used only for a prototype version of the widget. The code is tagged with `// PROTOTYPE` comments.
+
+Find more details about the prototype in this [proposal for a device switch](https://github.com/digitalservicebund/useid-architecture/blob/main/research/device-switch/proposal-qr-code-based-device-switch-with-webauthn.md).
```

Comment:
Which local file do you mean?
Since we do not need a specific version of that proposal, I found linking to the latest version appropiate, so we don't need to update the link when updating the file. If it might get removed, a permalink would be better. But I'm also fine with using a permalink, since we don't know if and when it'll be updated anyway. 🙃

Reply:
Oh yeah, other repo. Local file does not make any sense, sorry. 😉 
I would vote for a permalink since moving or renaming the file would already break this link here.

Reply:
Good point. 👍